task/testmap.py

```diff
@@ -179,7 +180,7 @@
         "fedora-31@cockpit-project/cockpit",
         "fedora-31/selenium-firefox@cockpit-project/cockpit",
         "fedora-31/selenium-chrome@cockpit-project/cockpit",
-        "ubuntu-1804@cockpit-project/cockpit",
+        "ubuntu-2004@cockpit-project/cockpit",
```

Comment:
Why?

Reply:
Initially, because I'd like to drop the ubuntu-1804 image soon. But indeed this should be a separate PR. I'll revert that. Moreover, there's no freeipa on ubuntu-2004 ATM, so we couldn't test that part. I think we better test that on debian-stable then.